.storybook/webpack.config.js

```diff
@@ -13,8 +13,8 @@ module.exports = ({ config }) => {
   config.resolve.extensions.push('.ts', '.tsx');
 
   config.module.rules.push({
-    test: /stories.js$/u,
-    loaders: [require.resolve('@storybook/addon-storysource/loader')],
+    test: /stories.tsx$/u,
+    loader: require.resolve('@storybook/source-loader'),
```

Comment:
`@storybook/addon-storysource/loader` is [deprecated](https://github.com/storybookjs/storybook/blob/83ac3d95e5c993a780b44061e3e449b91e71d9d6/addons/storysource/loader.js) and the project needs to use `@storybook/source-loader` instead. 

> DeprecationWarning: @storybook/addon-storysource/loader is deprecated, please use @storybook/source-loader instead.

Reply:
Can this be removed from package.json?

Reply:
I think it does not have to be removed from the package.json because `storysource` is still used, but the loader is now located in a different place (at `@storybook/source-loader`).